phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -39,7 +39,11 @@
 
     @Bean
     public ViewLibrary viewLibrary() {
-        return new ViewLibrary(applicationName, themeConfiguration.themeManager(), phoenicisGlobalConfiguration.objectMapper());
+        return new ViewLibrary(
```

Comment:
The maven formatter plugin is messing this. Do you know how we could improve it?